src/main/java/net/openhft/chronicle/queue/QueueSystemProperties.java

```diff
@@ -35,7 +35,7 @@ private QueueSystemProperties() {
      * Default unset value: false
      * Activation values  : "", "yes", or "true"
      */
-    public static final boolean CHECK_INDEX = Jvm.getBoolean("queue.check.index");
+    public static boolean CHECK_INDEX = Jvm.getBoolean("queue.check.index");
```

Comment:
We need to be able to disable this in order for PartialUpdateTest to replicate what happens in "prod"